test/helper/test_timeline.ts

```diff
@@ -110,7 +110,7 @@ class TimelineParserLint extends TimelineParser {
 
     // Capture each part of the line (separated by spaces).
     // Anything encapsulated by double-quotes or braces will be treated as a single element.
-    const lineParts = line.match(/"[^"]*"|\{[^}]*\}|[^ ]+/g);
+    const lineParts = line.match(/"[^"]*"|\{.*\}|[^ ]+/g);
```

Comment:
Yeah, in this case the regex would break if we ever added another `{ ... }` object to the timeline structure. The only other way I could think of to match this would be to just find the first `{`, find the last `}`, and then keep trying to exec as JS to find the actual correct ending `}`.